job-store-service/war/src/test/java/dk/dbc/dataio/jobstore/service/ejb/JobSchedulerTransactionsBeanIT.java

```diff
@@ -23,41 +24,28 @@ public void findChunksToWaitFor() throws Exception {
 
         DependencyTrackingService service = new DependencyTrackingService().init();
 
-        assertThat(service.findChunksToWaitFor(new DependencyTracking(new TrackingKey(0, 0), 0)
-                        .setSubmitter(123456)
-                        .setMatchKeys(Collections.emptySet()), null),
-                is(Collections.emptySet()));
+        assertThat(service.findChunksToWaitFor(new DependencyTracking(new TrackingKey(0, 0), 0, 123456), null), is(Collections.emptySet()));
 
-        assertThat(service.findChunksToWaitFor(new DependencyTracking(new TrackingKey(0, 0), 0)
-                        .setSubmitter(123456)
-                        .setMatchKeys(asSet("K1")), null),
+        assertThat(service.findChunksToWaitFor(new DependencyTracking(new TrackingKey(0, 0), 0, 123456, of("K1")), null),
```

Comment:
> of("K1")

havde været mere læsliget for mig hvis det havde været 

>   Set.of("K1")

Så havde jeg søgt på of og fundet java.util.Set.of; 
Bør rettes til _Set.of_

Reply:
ikke enig.